### PR TITLE
fix: Disable copy and paste of ConfirmPassword at wallet creation

### LINKED ISSUE #856: Disable copy and paste of password in Wallet creation
When creating a wallet you are prompted with a password selection screen.
This screen has 2 inputs. 
- 1 to enter the password that you want and 
- underneath another to confirm the password

The confirmation input allows you to paste from the clipboard.
This means you may typo a password, copy and paste it into the confirmation input and then continue setting up the wallet leading to being unable to unlock the wallet in the future.

### PR DESCRIPTION
**Pull request by Another-DevX**
close #856

### CODE REVIEW

File: packages/app/src/systems/SignUp/components/CreatePassword/CreatePassword.tsx [hunk 1]
```diff
@@ -120,6 +120,7 @@ export function CreatePassword({
                   autoComplete="new-password"
                   placeholder="Confirm your password"
                   aria-label="Confirm Password"
+                  onPaste={(e) => e.preventDefault()}
```
Comment (LuizAsFight) [inline packages/app/src/systems/SignUp/components/CreatePassword/CreatePassword.tsx:123]: thanks for your contribution

this behavior is helpful for tests/development although I agree with the concern 
can you do a conditional using `IS_DEVELOPMENT` to only prevent when not in development mode ?

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/SignUp/components/CreatePassword/CreatePassword.tsx (hunk 1 above)
Comment (Another-DevX) [reply packages/app/src/systems/SignUp/components/CreatePassword/CreatePassword.tsx]: Change implemented

**Review by Another-DevX — COMMENTED** [review]
The change has been implemented

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
development = should allow copy/paste
production = should not allow copy/paste

**Comment by luizstacio** [comment]
@Another-DevX can you add a changeset for this PR `pnpm changeset add --empty`

**Comment by Another-DevX** [comment]
> @Another-DevX can you add a changeset for this PR `pnpm changeset add --empty`

Added!

**Review by helciofranco — APPROVED** [review]

**Pull request merged**